src/main/java/nextstep/config/WebMvcConfiguration.java

```diff
@@ -12,14 +15,23 @@
 public class WebMvcConfiguration implements WebMvcConfigurer {
 
     private final AuthenticationPrincipalArgumentResolver authenticationPrincipalArgumentResolver;
+    private final LoginService loginService;
 
-    public WebMvcConfiguration(AuthenticationPrincipalArgumentResolver authenticationPrincipalArgumentResolver) {
+    public WebMvcConfiguration(AuthenticationPrincipalArgumentResolver authenticationPrincipalArgumentResolver, LoginService loginService) {
         this.authenticationPrincipalArgumentResolver = authenticationPrincipalArgumentResolver;
+        this.loginService = loginService;
     }
 
     @Override
     public void addArgumentResolvers(List<HandlerMethodArgumentResolver> resolvers) {
         resolvers.add(authenticationPrincipalArgumentResolver);
     }
 
+    @Override
+    public void addInterceptors(InterceptorRegistry registry) {
+        registry.addInterceptor(new LoginInterceptor(loginService))
+                .excludePathPatterns("/members", "/login/token");
+        registry.addInterceptor(new AdminInterceptor())
+                .addPathPatterns("/admin/**");
```

Comment:
여러 개의 인터셉터를 등록한다면 순서를 명시적으로 표시해주시는 것이 좋습니다!

```suggestion
        registry.addInterceptor(new LoginInterceptor(loginService))
                .order(1)
                .excludePathPatterns("/members", "/login/token");
        registry.addInterceptor(new AdminInterceptor())
                .order(2)
                .addPathPatterns("/admin/**");
```